stripe/src/main/java/com/stripe/android/model/ModelUtils.java

```diff
@@ -28,36 +26,36 @@ static boolean isWholePositiveNumber(@Nullable String value) {
      *
      * @param year the input year, as a two or four-digit integer
      * @param month the input month
-     * @return {@code true} if the input time has passed, {@code false} otherwise.
+     * @param now the current time
+     * @return {@code true} if the input time has passed the specified current time,
+     *  {@code false} otherwise.
      */
     @SuppressWarnings("WrongConstant")
-    static boolean hasMonthPassed(int year, int month) {
-        if (hasYearPassed(year)) {
+    static boolean hasMonthPassed(int year, int month, Calendar now) {
+        if (hasYearPassed(year, now)) {
             return true;
         }
 
-        Calendar now = Clock.getCalendarInstance();
         // Expires at end of specified month, Calendar month starts at 0
-        return normalizeYear(year) == now.get(Calendar.YEAR)
+        return normalizeYear(year, now) == now.get(Calendar.YEAR)
                 && month < (now.get(Calendar.MONTH) + 1);
     }
 
     /**
      * Determines whether or not the input year has already passed.
      *
      * @param year the input year, as a two or four-digit integer
-     * @return {@code true} if the year has passed, {@code false} otherwise.
+     * @param now, the current time
+     * @return {@code true} if the input year has passed the year of the specified current time
+     *  {@code false} otherwise.
      */
-    static boolean hasYearPassed(int year) {
-        int normalized = normalizeYear(year);
-        Calendar now = Clock.getCalendarInstance();
+    static boolean hasYearPassed(int year, Calendar now) {
+        int normalized = normalizeYear(year, now);
         return normalized < now.get(Calendar.YEAR);
     }
 
-    // Convert two-digit year to full year if necessary
-    private static int normalizeYear(int year)  {
+    private static int normalizeYear(int year, Calendar now)  {
```

Comment:
As a side effect of this pass-through, we can now more interestingly test this function if we bump it from private to package-private. (another diff for another time)